### DISCUSSION TITLE
Area calculation by laser points?

### DISCUSSION THREAD

**Discussion by MonaFuhrmann** [biigle/core#619] [Q&A] [answered]
I wondered if somebody can shed light on how exactly the image area is calculated in Biigle? I have performed laser annotation, and compared given image area ( in Biigle) with calculated area from x,y coordinates - and they are slightly different. See quick screenshot from excel (output from image with laser distance 8 cm, 1920x1088 pixel, Biigle output in orange). Am I missing something here?
![Skjermbilde](https://github.com/biigle/core/assets/79697804/0ed0062a-d3f7-4fad-9a0c-cfb52c1d67fe)

**Comment by mzur** [comment, marked as answer]
You seem to calculate the laser distance only from the difference in x coordinates (in 1D). BIIGLE calculates the euclidean distance between the (x,y) points (in 2D). For the first pair of points, if you calculate √((1334.06 - 1244.06)² + (365.19 - 380.19)²) then you get a laser distance of 91.24 px. With a laser distance of 0.08 m this is 0.000876795 m/px and 0.000000769 m²/px. With 1920x1088 px you get 1.605928676 m²  (1.61 m²).

**Reply by MonaFuhrmann** [reply]
I see, this makes sense! Thanks
